projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -481,6 +489,14 @@ function ( $newsletter_category ) {
 						'jetpack_comment_form_color_scheme' => (string) get_option( 'jetpack_comment_form_color_scheme' ),
 						'in_site_migration_flow'           => (string) get_option( 'in_site_migration_flow', '' ),
 						'migration_source_site_domain'     => (string) get_option( 'migration_source_site_domain' ),
+						'jetpack_waf_automatic_rules'      => (bool) get_option( 'jetpack_waf_automatic_rules' ),
+						'jetpack_waf_ip_allow_list'        => (string) get_option( 'jetpack_waf_ip_allow_list' ),
+						'jetpack_waf_ip_allow_list_enabled' => (bool) get_option( 'jetpack_waf_ip_allow_list_enabled' ),
+						'jetpack_waf_ip_block_list'        => (string) get_option( 'jetpack_waf_ip_block_list' ),
+						'jetpack_waf_ip_block_list_enabled' => (bool) get_option( 'jetpack_waf_ip_block_list_enabled' ),
+						'jetpack_waf_share_data'           => (bool) get_option( 'jetpack_waf_share_data' ),
+						'jetpack_waf_share_debug_data'     => (bool) get_option( 'jetpack_waf_share_debug_data' ),
+						'jetpack_waf_automatic_rules_last_updated_timestamp' => (int) get_option( 'jetpack_waf_automatic_rules_last_updated_timestamp' ),
```

Comment:
This may in fact be auto-formatted/fixed as per the linter (I also see it was always previously the case), but the nitpicker inside me would love to see these all aligned nicely. A number of other changes that involve arrays here also have various misalignments between key/values pairs.